styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+}
+
+#main {
+  background-color: #eee;
+}
+
+header {
+  margin-top: 10px;
+  text-align: center;
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+#img-div {
```

Comment:
You should limit the usage of IDs, it is preferrable to use classes instead of IDs except in specific cases